ext/gh-droplets.json

```diff
@@ -1,1693 +1,1693 @@
 {
```

Comment:
what is this file?

Reply:
It's generated by `ext-inventory.yml` and it has a dump of all the machines running in our greenhost account.